tokens/components/bridge/stepper.css

```diff
@@ -37,15 +37,11 @@
 	--spectrum-stepper-border-radius: var(--system-stepper-border-radius);
 	--spectrum-stepper-button-width: var(--system-stepper-button-width);
 	--spectrum-stepper-button-padding: var(--system-stepper-button-padding);
-	--spectrum-stepper-width: var(--system-stepper-width);
```

Comment:
Similar to the coach indicator bride file, is this `--spectrum-stepper-width` really supposed to get removed from this file? I understand the remaining properties in this file since they're all `--mod` prefixed, but this one caught my eye as questionable.